### DISCUSSION TITLE
Make "shadow" utility works like "border" does

### DISCUSSION THREAD

**Discussion by mockingjet** [tailwindlabs/tailwindcss#4867] [Ideas]
For example,

```
class="border border-${color}"

# shadow can be used like this

class="shadow shadow-${color}"
```

This can apply in many aspects such as width, directions... etc.

**Comment by olets** [comment]
This is a good use case for a custom plugin. And because it's a small plugin it will be a good starting point for you experiment with adding your own "many aspects such as width, directions… etc" enhancements.

https://play.tailwindcss.com/GKTthSOKpG?file=config
